src/30_Advanced/Autosuggest.html

```diff
@@ -12,22 +12,25 @@
 -->
 <!--
 ## Setup
-
-First, the markup declares the page as AMP, contains the AMP boilerplate, and
-includes the required AMP extensions.
 -->
+<!-- -->
 <!doctype html>
 <html ⚡>
 <head>
   <title>Autosuggest form field example</title>
   <meta charset="utf-8">
   <script async src="https://cdn.ampproject.org/v0.js"></script>
-  <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+  <!-- First we include `amp-form` -->
```

Comment:
If it makes sense it'd be nice to explain why we need the respective components.